src/main/java/com/epam/controller/IndexController.java

```diff
@@ -1,19 +1,34 @@
 package com.epam.controller;
 
+import com.epam.model.User;
+import com.epam.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
 
-import java.time.LocalDateTime;
-import java.util.HashMap;
-import java.util.Map;
 
 @Controller
+@RequestMapping(path="/demo")
 public class IndexController {
-    @GetMapping("/")
-    public ModelAndView index() {
-        Map<String, String> model = new HashMap<>();
-        model.put("name", "Alexey");
-        return new ModelAndView("index", model);
+    @Autowired
+    private UserRepository userRepository;
```

Comment:
Always prefer all args contructor DI (don't use Autowired)

Reply:
There is a paragraph called "Constructor-based or setter-based DI?" at the following link to Spring Docs which briefly explains why using constructor is the preferable way to inject your dependencies.
https://docs.spring.io/spring/docs/current/spring-framework-reference/core.html#beans-setter-injection